LOAN/RealEstateLoans/HomeMortgageDisclosureActCoveredMortgages.rdf

```diff
@@ -87,66 +87,66 @@
 		<skos:definition>category based on a cultural factors, including nationality, regional culture, ancestry, and language</skos:definition>
```

Comment:
Any human has an ethnicity surely

Reply:
@rivettp - I believe this was developed when MU was working on the ontology for LOAN - needs an SME to review the entire ontology in a new issue